Proteomics/ModificationWithLocation.cs

```diff
@@ -64,20 +64,11 @@ public override bool Equals(object o)
             return m == null ? false :
 
                base.Equals(m)
-               && (this.accession == null && m.accession == null
```

Comment:
Okay, that makes sense. We are considering modifications to be unique chemical entities.